docs/src/index.md

```diff
@@ -2,9 +2,89 @@
 CurrentModule = QuantumToolbox
 ```
 
-# Introduction
+# [Introduction](@id doc: Introduction)
 
-[QuantumToolbox](https://github.com/qutip/QuantumToolbox.jl) is a Julia package for numerical simulation of quantum systems.
+[QuantumToolbox.jl](https://github.com/qutip/QuantumToolbox.jl) is a cutting-edge Julia package designed for quantum physics simulations, closely emulating the popular Python [QuTiP](https://github.com/qutip/qutip) package. It uniquely combines the simplicity and power of Julia with advanced features like GPU acceleration and distributed computing, making simulation of quantum systems more accessible and efficient.
 
-```@index
+*With this package, moving from Python to Julia for quantum physics simulations has never been easier*, due to the similar syntax and functionalities.
+
+## Features
+
+QuantumToolbox.jl is equipped with a robust set of features:
+
+- **Quantum State and Operator Manipulation:** Easily handle quantum states and operators with a rich set of tools, with the same functionalities as QuTiP.
+- **Dynamical Evolution:** Advanced solvers for time evolution of quantum systems, thanks to the powerful [DifferentialEquations.jl](https://github.com/SciML/DifferentialEquations.jl) package.
+- **GPU Computing:** Leverage GPU resources for high-performance computing. For example, you run the master equation direclty on the GPU with the same syntax as the CPU case.
+- **Distributed Computing:** Distribute the computation over multiple nodes (e.g., a cluster). For example, you can run undreds of quantum trajectories in parallel on a cluster, with, again, the same syntax as the simple case.
+- **Easy Extension:** Easily extend the package, taking advantage of the Julia language features, like multiple dispatch and metaprogramming.
+
+## Brief example
```

Comment:
Maybe make this `Brief example` as a new page in `Front Matter` (after `Installation` or `Key differences from QuTiP`) and title as `Quick Start` ?
Cause I think it is better to follow the installation first and then know how to use the code.
What do you think?

Reply:
I don’t know if moving to a new page is good. For sure we can change the order and put the installation instruction first. Because I think that it is important that the reader knows the simplicity of moving from QuTiP, already in the first page.